README.md

```diff
@@ -29,25 +29,42 @@ func main() {
 	config := flow.Config{
 		CPUCoresNumber: 8,
 	}
-	flow.SystemInit(&config)
+	if err := flow.SystemInit(&config); err != nil {
```

Comment:
I think you should wrap this into universal error handling function: CheckFatal(flow.SystemInit())